testsuite/ext-tools/maintenance_json_generator.py

```diff
@@ -77,10 +77,12 @@
 nodesdict42 = {
     "server": ["/SUSE_Updates_SLE-Module-SUSE-Manager-Server_4.2_x86_64/",
                "/SUSE_Updates_SLE-Product-SUSE-Manager-Server_4.2_x86_64/",
+               "SUSE_Updates_SLE-Product-SLES_15-SP3-LTSS_x86_64/",
```

Comment:
Is there a `/` missing at the beginning of all new entries?

Reply:
Yes indeed, thanks, fixed now